[2025-10-09T04:51:10.734+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: order_generator.load_orders scheduled__2025-10-06T19:00:00+00:00 [queued]>
[2025-10-09T04:51:10.753+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: order_generator.load_orders scheduled__2025-10-06T19:00:00+00:00 [queued]>
[2025-10-09T04:51:10.754+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2025-10-09T04:51:10.754+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 2
[2025-10-09T04:51:10.755+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2025-10-09T04:51:10.773+0000] {taskinstance.py:1304} INFO - Executing <Task(PythonOperator): load_orders> on 2025-10-06 19:00:00+00:00
[2025-10-09T04:51:10.779+0000] {standard_task_runner.py:55} INFO - Started process 2369 to run task
[2025-10-09T04:51:10.783+0000] {standard_task_runner.py:82} INFO - Running: ['airflow', 'tasks', 'run', 'order_generator', 'load_orders', 'scheduled__2025-10-06T19:00:00+00:00', '--job-id', '100', '--raw', '--subdir', 'DAGS_FOLDER/order_loader_dag.py', '--cfg-path', '/tmp/tmp4grdb3hr']
[2025-10-09T04:51:10.787+0000] {standard_task_runner.py:83} INFO - Job 100: Subtask load_orders
[2025-10-09T04:51:10.911+0000] {task_command.py:389} INFO - Running <TaskInstance: order_generator.load_orders scheduled__2025-10-06T19:00:00+00:00 [running]> on host 81fbc710bbce
[2025-10-09T04:51:11.001+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=order_generator
AIRFLOW_CTX_TASK_ID=load_orders
AIRFLOW_CTX_EXECUTION_DATE=2025-10-06T19:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2025-10-06T19:00:00+00:00
[2025-10-09T04:51:11.028+0000] {logging_mixin.py:137} INFO - Successfully loaded data to orders
[2025-10-09T04:51:11.238+0000] {logging_mixin.py:137} INFO - Successfully loaded data to order_items
[2025-10-09T04:51:11.239+0000] {python.py:177} INFO - Done. Returned value was: None
[2025-10-09T04:51:11.252+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=order_generator, task_id=load_orders, execution_date=20251006T190000, start_date=20251009T045110, end_date=20251009T045111
[2025-10-09T04:51:11.277+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2025-10-09T04:51:11.294+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
